src/api/__tests__/wampApi.spec.ts

```diff
@@ -1,10 +1,37 @@
 import {WampApi} from '../wampApi';
 
-const wamp = new WampApi();
-
 describe('wamp service', () => {
-  it('wamp session should be defined after connection established', () => {
-    expect(wamp.connect).toBeDefined();
-    expect(true).toBe(true);
+  let wamp: WampApi;
+
+  describe('wamp', () => {
+    beforeEach(() => {
+      wamp = new WampApi();
+      wamp.throttle = jest.fn();
+    });
+
+    it('session should be defined after connection established', async () => {
+      expect(wamp.connect).toBeDefined();
```

Comment:
Not sure that it make sense to insert toBeDefined everywhere. Not even sure it's required